fvm/src/machine/default.rs

```diff
@@ -56,7 +57,7 @@ where
         circ_supply: TokenAmount,
         network_version: NetworkVersion,
         state_root: Cid,
-        builtin_actors: Cid,
+        builtin_actors: (u32, Option<Cid>),
```

Comment:
Just to confirm, passing this version number here is temporary to avoid breaking the tests? A follow-up PR will entirely remove the version number here? (Note that this affects the FFI layer too)

Reply:
for now yes, and we can certainly remove it once we have proper v1 bundles.

We might want to keep it for future proofing, but that is not rewuired given that there is no (and wont be for quite a while) version other than v1.

Reply:
The reason I ask is that the version _really_ doesn't go here, at all.

- We only need it now to deal with the temporary situation of the schema change during development.
- In the future, if there are any System Actor schema changes, they would go coupled with a new network version.
- The builtin_actors CId argument is only here to support the canary mode (nv14 and nv15, where the system actor doesn't hold the manifest). It might eventually go as well.

Can you submit the follow-up PR to erase this version? We can make a release with only this PR, then release builtin-actors, then merge the new PR.

Reply:
one intermediate step is making the bundles, but yes of course i will make that pr.

Reply:
agreed in either case

Reply:
follow up in #404